### PR TITLE
chore: testing

### PR DESCRIPTION
**Pull request by jinqi7**
For testing

### CODE REVIEW

File: pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml [hunk 1]
```diff
@@ -241,51 +241,6 @@ spec:
           workspace: release-workspace
       runAfter:
         - push-snapshot
-    - name: push-sbom-to-pyxis
```
Comment (scoheb) [inline pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml]: why are you removing this? not part of your FBC work I think

Comment (johnbieren) [reply pipelines/rh-push-to-external-registry/rh-push-to-external-registry.yaml]: PR description says it is just for testing and not merging

**Review by johnbieren — CHANGES_REQUESTED** [review]
I no longer see that this PR is just for testing and not merging but we will not merge a change into the development branch that changes the taskGitRevision to staging

**Comment by jinqi7** [comment]
> I no longer see that this PR is just for testing and not merging but we will not merge a change into the development branch that changes the taskGitRevision to staging

I want to change the revision since the bug [RHTAPREL-787](https://issues.redhat.com/browse/RHTAPREL-787) exists currently.

**Comment by johnbieren** [comment]
> > I no longer see that this PR is just for testing and not merging but we will not merge a change into the development branch that changes the taskGitRevision to staging
> 
> I want to change the revision since the bug [RHTAPREL-787](https://issues.redhat.com/browse/RHTAPREL-787) exists currently.

The taskGitRevision default value will match the current branch name for our pipelines without exception

**Comment by jinqi7** [comment]
> > > I no longer see that this PR is just for testing and not merging but we will not merge a change into the development branch that changes the taskGitRevision to staging
> > 
> > 
> > I want to change the revision since the bug [RHTAPREL-787](https://issues.redhat.com/browse/RHTAPREL-787) exists currently.
> 
> The taskGitRevision default value will match the current branch name for our pipelines without exception

Where the configuration can be changed to have the same effect instead of changing the value of revision?

**Comment by johnbieren** [comment]
> > > > I no longer see that this PR is just for testing and not merging but we will not merge a change into the development branch that changes the taskGitRevision to staging
> > > 
> > > 
> > > I want to change the revision since the bug [RHTAPREL-787](https://issues.redhat.com/browse/RHTAPREL-787) exists currently.
> > 
> > 
> > The taskGitRevision default value will match the current branch name for our pipelines without exception
> 
> Where the configuration can be changed to have the same effect instead of changing the value of revision?

If you want to use the staging version of the fbc-release pipeline, you should point your RPA at the staging branch of the release-service-catalog

**Comment by jinqi7** [comment]
> > > > > I no longer see that this PR is just for testing and not merging but we will not merge a change into the development branch that changes the taskGitRevision to staging
> > > > 
> > > > 
> > > > I want to change the revision since the bug [RHTAPREL-787](https://issues.redhat.com/browse/RHTAPREL-787) exists currently.
> > > 
> > > 
> > > The taskGitRevision default value will match the current branch name for our pipelines without exception
> > 
> > 
> > Where the configuration can be changed to have the same effect instead of changing the value of revision?
> 
> If you want to use the staging version of the fbc-release pipeline, you should point your RPA at the staging branch of the release-service-catalog

Currently, only "main" branch works well.  I can't identify the root cause.

**Comment by johnbieren** [comment]
> > > > > > I no longer see that this PR is just for testing and not merging but we will not merge a change into the development branch that changes the taskGitRevision to staging
> > > > > 
> > > > > 
> > > > > I want to change the revision since the bug [RHTAPREL-787](https://issues.redhat.com/browse/RHTAPREL-787) exists currently.
> > > > 
> > > > 
> > > > The taskGitRevision default value will match the current branch name for our pipelines without exception
> > > 
> > > 
> > > Where the configuration can be changed to have the same effect instead of changing the value of revision?
> > 
> > 
> > If you want to use the staging version of the fbc-release pipeline, you should point your RPA at the staging branch of the release-service-catalog
> 
> Currently, only "main" branch works well. I can't identify the root cause.

I mean, we have customers using other branches besides main so others work too. But we do need to find out what is wrong with the test for other branches. If you are just using this PR to pair with an e2e-tests PR to run the fbc tests, I am not sure why you are removing pieces of the fbc-release pipeline in it. Why not just a noop change? I am not surprised that removing parts of the fbc-release pipeine yaml is leading to issues... the lines you are removing in this PR have a purpose

**Comment by jinqi7** [comment]
> I mean, we have customers using other branches besides main so others work too. But we do need to find out what is wrong with the test for other branches. If you are just using this PR to pair with an e2e-tests PR to run the fbc tests, I am not sure why you are removing pieces of the fbc-release pipeline in it. Why not just a noop change? I am not surprised that removing parts of the fbc-release pipeine yaml is leading to issues... the lines you are removing in this PR have a purpose

It's commented out for another issue that  the tasks "sign-index-image" & "publish-index-image" can't be triggered as expected.

**Pull request closed without merging**